### PR TITLE
CIV-8884 - CUI: create and link access roles

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-8884

### Change description ###
Added 2 new Case roles.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: commons-civil/src/main/java/uk/gov/hmcts/reform/civil/enums/CaseRole.java [hunk 1]
```diff
@@ -7,7 +7,9 @@ public enum CaseRole {
     CREATOR,
     APPLICANTSOLICITORONE,
     RESPONDENTSOLICITORONE,
-    RESPONDENTSOLICITORTWO;
+    RESPONDENTSOLICITORTWO,
+    Claimant,
```
Comment (kenneth-hmcts) [inline commons-civil/src/main/java/uk/gov/hmcts/reform/civil/enums/CaseRole.java:11]: Is it CLAIMANT and DEFENDANT ?

Comment (TatsyMaslova) [inline commons-civil/src/main/java/uk/gov/hmcts/reform/civil/enums/CaseRole.java:11]: This should all be capital letters

**Comment by TatsyMaslova** [comment]
if you are making changes in assigning case to defendant in this story. have you checked that you can see the assigned case in dashboard? when going through the route pin and post / first contact

File: commons-civil/src/main/java/uk/gov/hmcts/reform/civil/enums/CaseRole.java (hunk 1 above)
Comment (neeta-hmcts) [reply commons-civil/src/main/java/uk/gov/hmcts/reform/civil/enums/CaseRole.java]: done

**Comment by neeta-hmcts** [comment]
> if you are making changes in assigning case to defendant in this story. have you checked that you can see the assigned case in dashboard? when going through the route pin and post / first contact

Yes.. done basic testing and can access the claim's from CUI.

**Review by sankhajuria — DISMISSED** [review]

**Review by TatsyMaslova — DISMISSED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Pull request merged**